src/deploy/campaign/environments.ts

```diff
@@ -88,6 +92,16 @@ export const getConfig = async ({
     zeroVaultAddressConf = process.env.ZERO_VAULT_ADDRESS;
   }
 
+  let zConfig : IZTokenConfig | undefined;
+  if (process.env.MOCK_Z_TOKEN === "true") {
```

Comment:
You don't ever need to test `if (A === true)`, in Typescript you can just do `if (A)`, or `if (!A)`. A third option would also be `if(!!A)` if you really wanted to get the truth value of something explicitly